### ISSUE TITLE
SNMP plugin not double-quoting strings and timeticks

### ISSUE THREAD

**Issue by mattbakerau** [influxdata/telegraf#1506]
I am trying to have Telegraf use the SNMP input plugin to capture some different metrics from an APC PDU.
Some of these are standard counters and they work fine, but other oids are either text or timetics and these seem to cause an error when the metric is entered into InfluxDB.

It seems that the SNMP plugin should be double-quoting values otherwise they are treated as boolean.

Using Telegraf & InfluxDB version 1.0.0 beta 2 on Centos 6.8

Config in telegraf.conf:
```
[[inputs.snmp]]

  [[inputs.snmp.host]]
    address = "10.1.1.3:161"
    community = "public"
    collect = ["PDUname"]

  [[inputs.snmp.get]]
        name = "PDUname"
        oid = ".1.3.6.1.2.1.1.5.0"
```

telegraf.log file:

2016/07/16 19:02:50 Output [influxdb] buffer fullness: 2 / 10000 metrics. Total gathered metrics: 2. Total dropped metrics: 0.
2016/07/16 19:02:50 ERROR: {"error":"unable to parse 'PDUname,host=monitor,snmp_host=10.1.1.3 PDUname=Rack 1468661560000000000': invalid boolean\nunable to parse 'PDUname,host=monitor,snmp_host=10.1.1.3 PDUname=Rack 1468661570000000000': invalid boolean"}
2016/07/16 19:02:50 Error writing to output [influxdb]: Could not write to any InfluxDB server in cluster

This example was for text OID. I found the same thing occurred if the OID was a timetics.

**Comment by Raczaron** [comment]
Hello,

I have exactly the same problem. Did you find a solution or a fix ?

**Comment by YBinnenweg** [comment]
I'm having the same issue while trying to parse diskIO metrics (diskIODevice). A fix would be highly appreciated! :)

**Comment by sparrc** [comment]
I believe that the new snmp plugin (https://github.com/influxdata/telegraf/pull/1389) should fix this. It will be available in release 1.0
